app/models/deleted_user.rb

```diff
@@ -1,2 +1,18 @@
 class DeletedUser < ApplicationRecord
+  def self.create_from_user(user)
+    return unless user
+
+    ActiveRecord::Base.transaction(requires_new: true) do
+      ::DeletedUser.create!(
```

Comment:
Would it make sense to call the class method directly? (I'm assuming this works)

```suggestion
      create!(
```

Reply:
yes 🤦 artifact of copy-pasting